agent/consul/health_endpoint.go

```diff
@@ -16,6 +17,19 @@ type Health struct {
 	srv *Server
 }
 
+type HealthGRPCAdapter struct {
+	Health
+}
+
+func (h *HealthGRPCAdapter) Stream(in *TestRequest, stream Health_StreamServer) error {
+	for {
+		if err := stream.Send(&TestReply{Data: int32(time.Now().Second())}); err != nil {
+			return err
+		}
+		time.Sleep(1 * time.Second)
+	}
+}
```

Comment:
This seems to be test-only code but it's not named that way which is confusing. Why do we need it? All streaming should go through the Consul.Subscribe endpoint right?

Reply:
Oh maybe this is old PoC code - can we kill it now?